packages/flutter_tts/CHANGELOG.md

```diff
@@ -4,4 +4,9 @@
 
 ## 1.0.1
 
-* Stabilize a speaking states management.
\ No newline at end of file
+* Stabilize a speaking states management.
+
+## 1.1.0
+
+* All APIs now return valid values.(0 or 1, empty list, etc.)
+    * Now the native implementation doesn't throw an error to dart
```

Comment:
Please refrain from using nested lists in the CHANGELOG file when you don't need to.